src/components/palettes.json

```diff
@@ -49,6 +49,15 @@
                         "Ctrl",
                         "Enter"
                     ]
+                },
+                {
+                    "id": "textInputBtn",
```

Comment:
Maybe it would be nice to add a shortcut to this button, as it might be commonly used?

Reply:
I will ask John in the issue